src/components/info/templates/default-template.vue

```diff
@@ -0,0 +1,152 @@
+<script setup lang="ts">
+import { defineProps } from 'vue'
+import {
+  FeatureInfoJSON,
+  FeatureJSON,
+} from '../../../services/info/feature-info.model'
+import { useTranslation } from 'i18next-vue'
+import {
+  hasAttributes,
+  isEmpty,
+  isLink,
+  showAttributesByLang,
+  hasValidFID,
+  prefixKeys,
+  getTrustedUrl,
+} from './template-utilities'
+import i18next from 'i18next'
+import ProfileFeatureInfo from '@/components/info/profile-feature-info.vue'
+
+defineProps({
+  layers: {
+    type: Object as () => FeatureInfoJSON,
+    required: true,
+  },
+})
+defineEmits<{
+  (e: 'export', payload: { feature: FeatureJSON; format: 'kml' | 'gpx' }): void
+}>()
+const { t } = useTranslation()
+const currentUrl = window.location.href
+</script>
+<template>
+  <div class="flex flex-col">
+    <div>
+      <h1 class="lux-poi-title" data-cy="defaultTemplateTitle">
+        {{ t(`${layers.layerLabel}`) }}
+      </h1>
+      <div
+        v-for="feature in layers.features"
+        :key="feature.id"
+        class="lux-poi-feature"
+      >
+        <h4 v-if="feature.attributes.label">
+          {{ t(`${feature.attributes.label}`) }}
+        </h4>
+        <div data-cy="defaultTemplateAttributes" v-if="hasAttributes(feature)">
+          <div
+            v-for="entry in prefixKeys(feature.attributes, 'f_').sort(
+              (a, b) => {
+                return layers.ordered ? 0 : a.key.localeCompare(b.key)
+              }
+            )"
+            :key="entry.key"
+          >
+            <span
```

Comment:
there is already a div wrapper above, i think you can get rid of this span (plus it is more logic to seperate lines with a div than with a span, not very important but this is html semantic)